platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -285,13 +282,15 @@ To counter this, platform teams should include product managers from the start
 to share roadmaps, gather feedback and generally understand and represent the
 needs of platform users.
 
-Another challenge when adopting platforms is choosing the right capabilities and
-experiences to enable first. Capabilities that are frequently required and
-undifferentiated, like pipelines, databases and observability, may be a good
-place to start. Platform teams may also choose to focus first on a limited
-number of engaged and skillful app teams. Detailed feedback from such teams
-improves the first platform experiences; and people from those teams help
-champion and evangelize the platform to later adopters.
+Additionally, Platform teams can easily be overloaded in their efforts to support many
```

Comment:
I really like this being moved down below the start of challenges. I think it reads really nicely as an addition to the challenges.

One concern I have though is if it ties into the three challenges that these paragraphs are trying to elaborate on. IMO "Platform teams must carefully choose their priorities and initial partner application teams" speaks to prioritising the roadmap, not the implementation.

Therefore, I would propose we keep the `## Enabling platform teams` header and move this section to actually after the whole `## Challenges with platforms`?

Reply:
@abangser does the below suggestion reflect what you mean?

```suggestion
## Enabling platform teams

Additionally, Platform teams can easily be overloaded in their efforts to support many
```

Reply:
That is definitely half of it. I was just wondering if that change plus the paragraph would be better suited under the `Finally, it's vital...` paragraph. As in, does this section fit better as a wrap up or in the middle 🤔

Reply:
Ah, I gotcha. Somehow I thought we were already at the end 😅 .

I agree, it would function nicely as a wrap-up. Which makes sense, since it was originally wrapping up the above section.
I'll send in another commit with some changed wording as new suggestion